app/react/Viewer/components/Viewer.js

```diff
@@ -243,4 +245,5 @@ const mapDispatchToProps = dispatch =>
     dispatch
   );
 
-export default connect(mapStateToProps, mapDispatchToProps)(Viewer);
+const ConnectedViewer = connect(mapStateToProps, mapDispatchToProps)(Viewer);
+export { ConnectedViewer, Viewer };
```

Comment:
If we are exporting the `Viewer` only for testing, it is better to just export the connected component as Viewer and use the `wrappedComponent` in the test.